$MirOS: ports/net/mcabber/patches/patch-src_histolog_c,v 1.2 2008/02/27 22:19:29 tg Exp $
--- src/histolog.c.orig	Thu Apr 17 19:28:38 2008
+++ src/histolog.c	Wed Jun 25 17:31:17 2008
@@ -349,7 +349,7 @@ void hlog_read_history(const char *bjid,
 }
 
 //  hlog_enable()
-// Enable logging to files.  If root_dir is NULL, then $HOME/.mcabber is used.
+// Enable logging to files.  If root_dir is NULL, then $HOME/.etc/mcabber is used.
 // If loadfiles is TRUE, we will try to load buddies history logs from file.
 void hlog_enable(guint enable, const char *root_dir, guint loadfiles)
 {
@@ -375,7 +375,7 @@ void hlog_enable(guint enable, const cha
       }
     } else {
       char *home = getenv("HOME");
-      const char *dir = "/.mcabber/histo/";
+      const char *dir = "/.etc/mcabber/histo/";
       RootDir = g_strdup_printf("%s%s", home, dir);
     }
     // Check directory permissions (should not be readable by group/others)
